guides/office/clear_desk_clear_screen.md

```diff
@@ -0,0 +1,24 @@
+# Clear Desk/Clear Screen
+
+Made Tech operates a Clear Desk/Clear Screen policy. This means that all sensitive/confidential materials should be removed from a user workspace and secured when no one is using it.
+
+## Clear Desk:
+- All ‘Confidential’ information must be hidden from view when not being actively used
+- Hard copies should be kept in locked drawers or returned to the appropriate place in the office 
+- Drawers/files containing sensitive information must be kept closed and locked. The keys to these drawers/files should be kept in a secure location and not left unattended at any time
+- Printouts should be immediately removed from the printer
+- When no longer needed, sensitive documents should be shredded in the office shredder
+- Passwords to confidential material should not be left on pieces of paper, sticky notes, or left unattended on a desk
+
+## Clear Screen:
+- Electronic copies of confidential materials should always be stored in the appropriate Made Tech tool (Google Team Drive, GitHub etc.)
```

Comment:
I think this is covered elsewhere. Unsure it is relevant to clear screen?

Reply:
removed 👍